t/venom/pfservers/configurator/step1.yml

```diff
@@ -0,0 +1,112 @@
+
+- name: step1_management_configuration
+  steps:
+  - type: http
+    method: PATCH
+    url: '{{.pfserver_webadmin_url}}/api/v1/configurator/config/interface/{{.configurator_management_interface}}'
+    ignore_verify_ssl: true
+    body: >-
+      {
+        "additional_listening_daemons": ["portal"],
+        "0": "portal",
+        "address": "{{.configurator_management_interface_ip_and_mask}}",
+        "high_availability": 0,
+        "hwaddr": "{{.configurator_management_mac}}",
+        "id": "{{.configurator_management_interface}}",
+        "ifindex": "2",
+        "ipaddress": "{{.configurator_management_ip}}",
+        "is_running": true,
+        "name": "{{.configurator_management_interface}}",
+        "netmask": "{{.configurator_management_mask}}",
+        "network": "{{.configurator_management_network_ip}}",
+        "network_iseditable": false,
+        "type": "management"
+      }
+    assertions:
+      - result.statuscode ShouldEqual 200
+
+- name: step1_registration_configuration
+  steps:
+  - type: http
+    method: PATCH
+    url: '{{.pfserver_webadmin_url}}/api/v1/configurator/config/interface/{{.configurator_registration_interface}}'
+    ignore_verify_ssl: true
+    body: >-
+      {
+        "dhcpd_enabled": "enabled",
+        "high_availability": 0,
+        "hwaddr": "{{.configurator_registration_mac}}",
+        "id": "{{.configurator_registration_interface}}",
+        "ifindex": "2",
+        "ipaddress": "{{.configurator_registration_ip}}",
+        "is_running": true,
+        "name": "{{{.configurator_registration_interface}}",
+        "netmask": "{{.configurator_registration_mask}}",
+        "network_iseditable": false,
+        "type": "vlan-registration"
+      }
+    assertions:
+      - result.statuscode ShouldEqual 200
+
+- name: step1_isolation_configuration
+  steps:
+  - type: http
+    method: PATCH
+    url: '{{.pfserver_webadmin_url}}/api/v1/configurator/config/interface/{{.configurator_isolation_interface}}'
+    ignore_verify_ssl: true
+    body: >-
+      {
+        "dhcpd_enabled": "enabled",
+        "high_availability": 0,
+        "hwaddr": "{{.configurator_isolation_mac}}",
+        "id": "{{.configurator_isolation_interface}}",
+        "ifindex": "7",
+        "ipaddress": "{{.configurator_isolation_ip}}",
+        "is_running": true,
+        "name": "{{.configurator_isolation_interface}}",
+        "netmask": "{{.configurator_isolation_mask}}",
+        "network_iseditable": false,
+        "type": "vlan-isolation"
+      }
+    assertions:
+      - result.statuscode ShouldEqual 200
+
+- name: step1_dns_configuration
+  steps:
+  - type: http
+    method: PUT
+    url: '{{.pfserver_webadmin_url}}/api/v1/configurator/config/system/dns_servers'
+    ignore_verify_ssl: true
+    body: >-
+      {
+        "dns_servers": ["{{.configurator_dns_ip}}", "{{.configurator_dns_aux_ip}}"],
+        "0": "{{.configurator_dns_ip}}",
+        "1": "{{.configurator_dns_aux_ip}}"
+      }
```

Comment:
This payload doesn't seem correct because you provide information twice.

Reply:
By removing dns_server or removing 0 and 1 it doesn't work

Reply:
There is certainly a reason to make things like this. Let's keep it like this.